android/src/app/organicmaps/maplayer/MapButtonsController.java

```diff
@@ -268,10 +268,14 @@ public float getBottomButtonsHeight()
 
   public void showMapButtons(boolean show)
   {
-    if (show)
+    if (show) {
       UiUtils.show(mFrame);
-    else
+      showButton(true, MapButtons.zoom);
+    }
+    else {
       UiUtils.hide(mFrame);
+      showButton(false, MapButtons.zoom);
```

Comment:
The zoom buttons are inside the frame, so there is no need to hide them if we hide the parent frame.